src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,77 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (changedPage: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage = 3,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const qtyBookmarks = Math.ceil(total / perPage);
+  const pages: number[] = [];
+
+  for (let i = 1; i <= qtyBookmarks; i += 1) {
+    pages.push(i);
+  }
```

Comment:
you can simplify it using the getNumbers function

Reply:
Hi! Hm, good idea! Done)